lib/srv/db/elasticsearch/engine.go

```diff
@@ -156,11 +163,19 @@ func copyRequest(ctx context.Context, req *http.Request) (*http.Request, error)
 // process reads request from connected elasticsearch client, processes the requests/responses and send data back
 // to the client.
 func (e *Engine) process(ctx context.Context, sessionCtx *common.Session, req *http.Request, client *http.Client) error {
-	reqCopy, err := copyRequest(ctx, req)
+	body, err := io.ReadAll(io.LimitReader(req.Body, teleport.MaxHTTPRequestSize))
 	if err != nil {
 		return trace.Wrap(err)
 	}
 
+	reqCopy, err := copyRequest(ctx, req, bytes.NewReader(body))
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	// audit log
```

Comment:
Can we remove this comment ? For me it looks superfluous where the below `emitAuditEvent` function call bring more value than this comment.